common/frame.go

```diff
@@ -1402,11 +1402,10 @@ func (f *Frame) WaitForLoadState(state string, opts goja.Value) {
 	f.log.Debugf("Frame:WaitForLoadState", "fid:%s furl:%q state:%s", f.ID(), f.URL(), state)
 	defer f.log.Debugf("Frame:WaitForLoadState:return", "fid:%s furl:%q state:%s", f.ID(), f.URL(), state)
 
-	rt := k6common.GetRuntime(f.ctx)
 	parsedOpts := NewFrameWaitForLoadStateOptions(f.defaultTimeout())
 	err := parsedOpts.Parse(f.ctx, opts)
 	if err != nil {
-		k6common.Throw(rt, fmt.Errorf("failed parsing options: %w", err))
+		k6Throw(f.ctx, "failed parsing wait for load state options: %v", err)
```

Comment:
I'm on the side of `cannot` and even without `cannot` or `failed` etc. But I'll stick w/`cannot`. There are a lot of other error messages spread in the codebase, but I won't touch them for now because they are out of the scope of this PR.

Reply:
https://github.com/grafana/xk6-browser/pull/211/files#diff-e29f62d5055939398c0185971827659487976598abd0b3cb0ad2a89152b2fe87L1409-R1408

Reply:
@imiric 

I think it's a good idea to have user-facing function names in error messages for more context.

So, what about prepending every throw with user-facing function's name first like:

`waitForNavigation: cannot parse options: %v`

or

`waitForLoadState: cannot parse options: %v`

etc.

I can make a PR after this one for every throw there.

Reply:
That LGTM, but it's not urgent. Maybe create an issue and we'll get to it at some point?

Reply:
Done: https://github.com/grafana/xk6-browser/issues/214